src/RuntimeContracts.Analyzer.Test/DoNotUseStandardContractAnalyzerTest.cs

```diff
@@ -98,38 +106,24 @@ class TypeName
         {
             public TypeName(string s)
             {
-                Contract.Requires(s != null);
+                [|Contract.Requires(s != null)|];
             }
         }
     }";
 
-            var diagnostic = GetFirstDiagnosticFor(test);
-            Assert.AreEqual(diagnostic.Id, DoNotUseStandardContractAnalyzer.DiagnosticId);
-
-            VerifyCSharpFix(test, test.Replace("System.Diagnostics.Contracts", "System.Diagnostics.ContractsLight"), codeFixIndex: null, allowNewCompilerDiagnostics: true);
-        }
-
-        protected override string GetAdditionalSources()
-        {
-            return @"
-namespace System.Diagnostics.ContractsLight
-{
-    public static class Contract
-    {
-        public static void Requires(bool predicate, string message = null) {}
-        public static void Assert(bool predicate, string message = null) {}
-    }
-}";
-        }
-
-        protected override CodeFixProvider GetCSharpCodeFixProvider()
-        {
-            return new RuntimeContractsAnalyzerCodeFixProvider();
-        }
-
-        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
-        {
-            return new DoNotUseStandardContractAnalyzer();
+            await new VerifyCS.Test
+            {
+                TestState =
+                {
+                    Sources = { test },
+                    ExpectedDiagnostics =
+                    {
+                        DiagnosticResult.CompilerError("CS1022").WithSpan(13, 1, 13, 2).WithMessage("Type or namespace definition, or end-of-file expected"),
```

Comment:
🐛 This is probably not what was expected, but I'm leaving it for a follow-up issue.

Reply:
➡️ Actually I went ahead and fixed this in 1089f25e04a0bc5b74df5b2bfd7838966cb90fc9